explanation/gpus-instances.md

```diff
@@ -1,38 +1,43 @@
+If the flag is used and there is no GPU available, Anbox Cloud uses software rendering.
+If the flag is not used, irrespective of GPU availability, Anbox Cloud uses the `null` platform and cannot perform any rendering. See {ref}`sec-null-platform` for the `null` platform configuration.
 
-See {ref}`sec-gpu-slots` for detailed information.
+## With a GPU
 
-## Using GPUs inside an instance
+If Anbox Cloud detects a GPU device during deployment, it configures the cluster to use the GPU. AMS configures each LXD instance to pass through a GPU device from the host. Currently, all GPUs that are available to a machine are passed to every instance that owns a GPU slot. For NVIDIA GPUs, LXD uses the [NVIDIA container runtime](https://github.com/NVIDIA/nvidia-container-runtime) to make the GPU driver of the host available to the instance.
 
-AMS configures each LXD instance to pass through a GPU device from the host. Currently, all GPUs that are available to a machine are passed to every instance that owns a GPU slot. For NVIDIA GPUs, LXD uses the [NVIDIA container runtime](https://github.com/NVIDIA/nvidia-container-runtime) to make the GPU driver of the host available to the instance.
+To use instances instances with a GPU,
 
-Check the list of supported GPUs ({ref}`sec-supported-gpus`) to see if Anbox Cloud includes a driver for your GPU device. If a GPU driver is available inside the instance, there are no further differences in how to use it in comparison to a regular environment. If no GPU driver is available, you must provide it through an addon.
+- Configure the number of available GPU slots on the node. This decides how many instances can run on a given node because GPUs have limited sharing capacity amongst multiple instances. See {ref}`sec-gpu-slots` for detailed information.
+- Check the list of supported GPUs ({ref}`sec-supported-gpus`) to see if Anbox Cloud includes a driver for your GPU device. If a GPU driver is available inside the instance, there are no further differences in how to use it in comparison to a regular environment. If no GPU driver is available, you must provide it through an addon.
+- Launch your instance with the `--enable-graphics` flag.
 
-If you want to let an application use the GPU (even if you are not interested in streaming the visual output), launch it with the `--enable-graphics` flag. With this flag, the command will launch the instance using the `webrtc` platform, which will automatically detect the underlying GPU and make use of it.
+(sec-sw-rendering-video-encoding)=
+## Without a GPU
 
-    amc launch --enable-graphics my-application
+Anbox Cloud provides a way to run entirely without GPU by using software rendering. You can enable software rendering and video encoding by launching your application with the `--enable-graphics` flag. When there is no GPU and this flag is enabled, Anbox Cloud falls back to using the `null` platform.
```

Comment:
Anbox Cloud provides a way to run entirely without GPU by using software rendering. You can ....
->
Anbox Cloud supports deployment in environments without GPUs by utilizing software or headless rendering. In such environments, when no GPU is available, you can ...

For the context, here we may need to focus on **a deployment without GPU**. Because in a deployment with GPU, an application with instance type `a2.3` or no gpu-slots is allocated, people can still enable software rendering by using
`amc launch --enable-graphis <app>` command.

Reply:
> For the context, here we may need to focus on a deployment without GPU. Because in a deployment with GPU, an application with instance type a2.3 or no gpu-slots is allocated, people can still enable software rendering by using
`amc launch --enable-graphis <app> command.`

Yes the part about with the GPU is covered in the earlier section.